contracts/DaiHrd.sol

```diff
@@ -569,28 +562,26 @@ contract DaiHrd is ERC777, MakerFunctions {
 	}
 
 	// Utility Functions
-	function convertAttodaiToAttodaiHrd(uint256 attodai, uint256 rontodaiPerPot ) internal pure returns (uint256 attodaiHrd) {
+	function calculatedChi() public view returns (uint256 rontodaiPerPot) {
+		// mirrors Maker's calculation: rmul(rpow(dsr, now - rho, ONE), chi);
+		return safeMul27(rpow(pot.dsr(), now - pot.rho(), 10 ** 27), pot.chi());
+	}
```

Comment:
Just moved this from below so all of the private functions are at the bottom, public functions are grouped above.

Reply:
I also added the original Maker formula to the comment for easy comparison during auditing.